frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segment_statistics_modal.tsx

```diff
@@ -87,65 +111,84 @@ export function SegmentStatisticsModal({
 }: Props) {
   const magInfo = getResolutionInfo(visibleSegmentationLayer.resolutions);
   const layersFinestResolution = magInfo.getFinestResolution();
-  const dataSetScale = Store.getState().dataset.dataSource.scale;
+  const dataSetScale = useSelector((state: OxalisState) => state.dataset.dataSource.scale);
+  const additionalCoordinates = useSelector(
+    (state: OxalisState) => state.flycam.additionalCoordinates,
+  );
+  const hasAdditionalCoords = hasAdditionalCoordinates(additionalCoordinates);
+  const additionalCoordinateStringForModal = getAdditionalCoordinatesAsString(
+    additionalCoordinates,
+    ", ",
+  );
   const dataSource = useFetch(
```

Comment:
should probably be renamed to `segmentStatisticsObjects` or something like this :smile:

Reply:
yes